src/services /video-trailer-error-handle.ts

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+import { videoTrailerData } from "../data/video-trailer.ts";
+
+export async function videoTrailerErrorHandle(): Promise<string> {
+    let video = '';
+    try {
+        video = await axios.get(videoTrailerData.videoUrl, { timeout: 1000 })
+    } catch (e) {
+        if (e) {
+            console.log('Video was not loaded')
+            video = videoTrailerData.posterUrl
+        }
+    }
+    return video
```

Comment:
I think that bad logic because we have mobile devices with unstable internet. On mobile devices in most cases this code will give me image instead of video url